Signal/src/ViewControllers/ConversationView/Cells/OWSMessageBubbleView.m

```diff
@@ -1312,6 +1312,7 @@ - (CGSize)measureSize
 
     if (self.hasBottomFooter) {
         CGSize footerSize = [self.footerView measureWithConversationViewItem:self.viewItem];
+        footerSize.width = MIN(footerSize.width, self.conversationStyle.maxMessageWidth);
```

Comment:
can this logic go into the footerView?

Reply:
I'll do this before I merge, but I don't want to conflict with unmerged PRs.